src/inc/til/winrt.h

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+namespace til // Terminal Implementation Library. Also: "Today I Learned"
+{
+    namespace details
+    {
+        template<typename T>
+        constexpr T winrt_empty_value() noexcept
+        {
+            if constexpr (std::is_base_of_v<winrt::Windows::Foundation::IUnknown, T>)
+            {
+                return nullptr;
+            }
+            else
+            {
+                return {};
+            }
+        }
+    };
+
+    template<typename T>
+    struct property
+    {
+        property<T>(const T& defaultValue = details::winrt_empty_value<T>()) :
+            _value{ defaultValue } {}

```

Comment:
```suggestion
        explicit constexpr property(auto&&... args) :
            _value{ std::forward<decltype(args)>(args)... } {}
```

This should most likely allow you to drop `winrt_empty_value`, as well as the `.c_str()` in `ControlProperties.h`.